static/locales/en.client.json

```diff
@@ -2160,5 +2165,19 @@
         "Preview not available.": "Preview not available.",
         "{{index}} of {{total}}": "{{index}} of {{total}}",
         "Uploading…": "Uploading…"
+    },
+    "AdminChecks": {
```

Comment:
Thanks for the change on the backticks :)

The locale file needs a little update now, to remove the `\n` from the generated stuff.